### DISCUSSION TITLE
OpenDTU zusammen mit DTU s lite betreiben

### DISCUSSION THREAD

**Discussion by turrican944** [tbnobody/OpenDTU#1218] [General]
Moin, 
es wird immer wieder berichtet das man auch mehrere DTUs mehr oder weniger gut parallel betreiben kann. Das habe ich mit einem HM600, OpenDTU und DTU Wlite ausprobiert geht.
Mein HMS1600 mit OpenDTU funktioniert, mit der original DTUs lite geht es auch. Zusammen funktioniert es aber überhaupt nicht, ich vermute die Frequenz in der OpenDTU ist falsch. Dort ist 865MHz eingestellt (war so der default wert und funktioniert). Die original DTU hat 868 MHz Autokanal 8. 
Gut in der OpenDTU 868 MHz eingestellt, funktioniert immer noch nicht wenn die Original DTU verbunden ist, schalte ich die aus verbindet sich OpenDTU irgendwann nach 10min. 
Gibts irgendwo eine Übersicht welcher Kanal welche Frequenz ist ?

**Comment by DPO99** [comment]
Nein das geht nicht. Niemals einen WR an 2 DTU betreiben.

**Reply by oldieman** [reply]
Geht doch. Ich habe hier bei mir 4 DTU an einem HM1500. Funktioniert ohne Probleme. Man sollte die Abfragezeit bei den DTUs nicht auf 1 Sekunde setzen. Alle 5 Sekunden ist auch genug.

**Reply by DPO99** [reply]
Mach das. 

1. Ist völlig sinnlos 
2. zeigt Dir im Zweifel nur Unsinn an 
3. sorgt ggf. dafür das die WR machen was Sie wollen

**Reply by stefan123t** [reply]
Also nochmal zum mitschreiben, es wird weder von Hoymiles noch von OpenDTU unterstützt/ empfohlen einen WR von zwei oder mehr DTUs gleichzeitig abzufragen.
Grund ist dass die DTU Anfragen stellt die teilweise aus mehreren Request Paketen bestehen und die meisten Antworten des WR ebenfalls aus mehreren Response Paketen.

Es gibt prinzipiell keine Arbitrierung bzw Prüfung durch die DTU oder den WR ob der Kanal zum Senden frei ist. D.h. die DTU ist immer alleine der Master und sendet wann es ihr passt / es per Intervall eingestellt ist, und die WR antworten sobald sie angesprochen werden brav darauf.

Wenn jetzt dabei die DTU 1 ein Antwort Paket nicht richtig empfangen hat dann fordert sie ggf eine Retransmission des Paketes erneut an. Wenn aber fast gleichzeitig die DTU 2 kurz vorher einen anderen Request abgesetzt hat dann kommt mindestens eine der beiden DTUs durcheinander und erhält keine passenden Antworten auf die Anfragen.

Je nach dem eingestellten Intervall der beiden DTUs führt das schneller oder langsamer zu Überschneidungen oder einem Timing-Problem.
Sowohl die OpenDTU/Ahoy als auch die Hoymiles DTUs verwenden zwar eine eigene DTU ID um zu senden, aber die WR erwarten idR nur von einer DTU angesprochen und abgefragt zu werden.

Zu empfehlen ist es jedenfalls nicht, da der Inverter bei unverständlichen Paketen bzw. fehlerhaften Zeitstempeln (Vor/Zurückdatierung des sekundengenauen Timestamps) in den Paketen in einen eigenen Fehlermodus geht und ggf bis zu 15 Minuten nicht mehr antwortet.
Die Hoymiles DTU hat zwar ein relativ langes Intervall von mW 15 Minuten daher kann das evtl. sogar parallel mit einer OpenDTU/Ahoy eine Weile lang klappen, bis sich die beiden überschneiden und es zu Fehlern kommt.

Dies ist mitunter eine der Ursachen für das Event 2 Time calibration das in solchen und anderen Timing kritischen Fällen auftritt.

**Comment by turrican944** [comment]
Moin
Das es sinnlos ist, ist mir klar. Das es mit der HM Serie geht weis ich , danach habe ich nicht gefragt. Da bei der HM Serie problemlos geht und beim HMS nicht, deshalb die Frage.

**Comment by BeginHere42** [comment]
Der Wlite S Stick scheint die Frequenz zu ändern, wenn der HMS durch Enegie vom Solarpanel aufwacht und die erste Verbindung hergestellt wird. Die open DTU stellt man ja auf eine Frequenz ein. Leider scheint der Wlite S Stick fast immer schneller zu sein.
Hatte mir die Mühe gemacht die Frequenz gesucht mit der der Wlite Stick kommuniziert mit dem HMS. Open  DTU drauf eingestellt und lief bis zum nächsten morgen. Dann wieder keine Verbindung HMS  und open DTU. Bin am  Überlegen, ob es funktionieren könnte der Wlite S Stick per Zeitschaltuhr später am Tag zuzuschalten.

**Comment by BeginHere42** [comment]
Bei mir funktioniert es wenn man beim Wlite s Stick in der Hoymiles APP die Frequenz  mit Kanal auf manuell stellt. Kanal 8 war bei mir 868 MHz.

**Reply by Kalle2018** [reply]
Wo stellt man das in der Hoymiles App denn ein? Ich habe die Stelle nicht gefunden.

**Comment by BeginHere42** [comment]
App starten. Keine Anlage auswählen. Unten in der Mitte das Quadrat (mit vier Vierteln) antippen. Beim Handy die Wlan Verbindung mit der DTU Lite S auswählen und verbinden. Zurück in der App Toolkit auswählen. Unten DTU Daten auswählen. Dann SUB-1G Einstellungen. Einstellungen vornehmen. Das wars. War doch ganz leicht ;-).

**Comment by BeginHere42** [comment]
Beim Handy die Wlan Verbindung mit der DTU Lite S auswählen und verbinden. Bedeutet nicht in der Hoymiles APP sondern in den Einstellungen des Handys die Wlan Verbindung zur DTU Lite S aufbauen.

**Reply by matsch111** [reply]
Bei mir hat es auch geklappt. Stick updaten, dann manuell Band 8 und Frequenz 868 einstellen, speichern. Topp.

**Reply by stefan123t** [reply]
Beim HM WR springen wir immer durch alle fünf Kanäle und versuchen die Verbindung überall.
Bei den HMS / HMT WR setzen wir dem WR einen festen Kanal / Frequenzband, wenn die DTU Pro/Lite S da (zufällig) einen anderen verwendet, kommt es natürlich zu Problemen.
Bei beiden sollte man die Abfrageintervalle nicht zu eng setzen, damit sie nach Möglichkeit nicht miteinander kollidieren.

@BeginHere42 & @matsch111 ich tippe mal dass es auch mit Kanal/Band 5 bzw. Frequenz 865 MHz fest in der DTU Lite/Pro S mit den Out-of-the-box Einstellungen von OpenDTU harmonieren und klappen sollte ?

**Comment by ms49434** [comment]
Ich habe gerade mal die Frequenzen gescannt, die die Hoymiles DTU-S verwendet, um mit meinen HMS Wechselrichtern zu kommunizieren.
In der Hoymiles Installer-App lässt sich der Kanal manuell zwischen 1 und 10 einstellen.
Bis jetzt war mir aber auch nicht klar, welchem Kanal welche Frequenz zugeordnet ist.
Unter Umständen lassen sich damit auch einige Kommunikationsprobleme erklären, bei denen sich die Frequenz nicht auf oder in der Nähe von den von Hoymiles definierten Kanälen liegt.
Es kann aber auch erklären, warum bei mir alles 1a auf 868,25 MHz funktioniert (Kanal 8):

Kanal    Frequenz (Mhz)
1. 863,00
2. 863,75
3. 864,50
4. 865,25
5. 866,00
6. 866,75
7. 867,50
8. 868,25
9. 869,00
10. 869,75

**Reply by ulli55** [reply]
Und hier mein ganz großer Dank, einen Monat endlose Suche endlich zu Ende.
Ich habe meinen Hoymiles Wechselrichter 800W 2t gegen einen 1600 4T getaucht und am Haus weitere Umbauten getätigt.
Ja und irgenwann sagten unsere Nachbarn unsere Klingel geht nicht mehr.
Wir haben 3 Stck Funkgongs Friedland Libra+ 868MHz seit 15 Jahren ohne Störung und leider auch vieles Andere.
Natürlich sucht man immer am logischten, aber meist verkehrtesten Ende zuerst.
Die Hoymiles DTU-WLite-S auf Kanal 2 umgestellt, wie beschrieben und die Welt war wieder heil.
MfG Danke

**Comment by refoscoi** [comment]
Kann es sein, dass die etwas an der App geändert haben? Ich finde unter Toolkit keine "DTU Daten", auch unter allen anderen Einstellungen sehe ich keine Möglichkeit, den Kanal einzustellen.

**Reply by BeginHere42** [reply]
Nur das Symbol hat sich geändert. Ist jetzt ein Schraubenschlüssel unten in der Mitte bei der App.

**Reply by BeginHere42** [reply]
Wenn man das Toolkit geöffnet hat, geht man in das Menü des Handys, wo man das Wlan auswählen kann. Dort wählt man die DTUL-XXXX aus. Dann zurück in die Hoymiles App und das Symbol der DTU Lite antippen. Dann nach ganz unten scrollen und auf DT-Daten tippen. Neuers Fenster unten Sub-1G-Einstellungen auswählen.
Gruß

**Reply by stefan123t** [reply]
@broth-itk kannst Du das auch mal mit Deiner DTU Pro S mittracen ?

Danke schon mal im Vorraus !

Sollte ja für die zehn von @ms49434 o.a. Kanäle ziemlich ähnlich sein.

Vielleicht kannst Du ja zwei/drei Kanäle (1., 2. und 10.) einstellen ?

**Comment by Tabaluga0815** [comment]
Ich hatte nach dem Tausch meines defekten HM1200 auf den neuen HMS1600 auch Probleme, die Daten gleichzeitig von der DTU S-Lite und der  Ahoy DTU zu empfangen.
Vielen Dank für den Tip, hat jetzt auch bei mir funktioniert.

**Comment by broth-itk** [comment]
@stefan123t Hier ist der Trace.

Habe zunächst auf Manuell gestellt, Kanal 1, dann auf Kanal 5, Kanal 10 und schließlich wieder auf "Auto"

[communication_log.txt](https://github.com/user-attachments/files/20653846/communication_log.txt)

**Reply by kernifpv** [reply]
Welche App bzw. welches Betriebssystem verwendet ihr? Android?
Ich frage, weil ich in der aktuellen iOS-App nach zwei Stunden Suchen den entsprechenden Punkt nicht finde.
Im Toolkit scheint es keine Option zu geben, mit der man den Kanal des Wechselrichters wechseln kann.
Danke!

**Reply by Jetiman** [reply]
Ich Android

**Reply by kernifpv** [reply]
hmm welche version, am handy von der frau seh ich diesbzeüglich auch nichts sehr eigenartig

**Reply by kernifpv** [reply]
ach ich war blind alles gut mit android gehts

**Comment by Onex1981** [comment]
Hallo 👋 Kann mir jemand helfen von euch finde keine Optionen zum einstellen bei DTU Daten.
Kann da nirgendwo hin klicken.
![IMG_20250902_205850](https://github.com/user-attachments/assets/25c05c5c-4845-47cc-b745-c8e19b5a8ff3)

**Reply by BeginHere42** [reply]
Sieht nicht wie die S-Miles Installer App aus. Falls es die S-Miles Home App ist, kann sein, dass es die Einstellung dort nicht gibt.

**Reply by Jetiman** [reply]
Also ich finde den Punkt grade auch nicht mehr. Es gab App Updates. Evtl haben die das entfernt?

Edit: Habe jetzt einmal die App v 3.2.1 installiert. Da gibt es die Option. Wenn man die aktuelle 3.4.0 installiert ist sie nicht vorhanden.

Apkpure com

**Reply by Onex1981** [reply]
Okay Danke 👍 für den Tip mit den App Versionen. 
Werd ich probieren wenn ich Zuhause bin.

**Reply by Onex1981** [reply]
Nachtrag:
Mit der App v 3.2.1 hab ich keine Verbindung zur DTU herstellen können steht irgendwas von Daten kompromittiert.
Hab dann App Version 3.3.0 probiert und da hat die Verbindung auf anhieb funktioniert und die Sub-1G-Einstellungen unter DTU Daten sind auch verfügbar somit alles Tip-top
